packages/swingset-runner/src/main.js

```diff
@@ -590,7 +591,7 @@ export async function main() {
     }
     const blockEndTime = readClock();
     if (forceGC) {
-      global.gc();
+      globalThis.gc();
```

Comment:
`globalThis.gc()` makes use of a powerful global that isn't declared along with `__dirname` above. So a future pass over our `/* global ... */` declarations won't find it. Options I can see:

 - Change this to `gc()` and declare `/* global __dirname, gc */`.
 - Remove `globalThis` from the list of globals in our eslint config above.

Reply:
I will proceed with the second. We might want to do the first as well, but not in this PR.

Thanks!